backend/zeno_backend/routers/sdk.py

```diff
@@ -86,6 +88,34 @@ async def verify_api_key(self, api_key: str) -> bool:
 router = APIRouter(tags=["zeno"], dependencies=[Depends(APIKeyBearer())])
 
 
+@router.get("/project-by-name/{owner_name}/{project_name}")
+async def get_project_by_name(
+    owner_name: str, project_name: str, api_key=Depends(APIKeyBearer())
+):
+    """Get a project by it's name and owner name.
+
+    Args:
+        owner_name (str): name of the project's owner.
+        project_name (str): name of the project.
+        api_key (_type_, optional): API key of the user making the request.
+            Defaults to Depends(APIKeyBearer()).
+
+    Returns:
+        str: uuid of the requested project.
+    """
+    uuid = await select.project_uuid(owner_name, parse.unquote(project_name))
+    if uuid is None:
+        raise HTTPException(
+            status_code=status.HTTP_400_BAD_REQUEST,
+            detail=(
+                "ERROR: Project " + owner_name + "/" + project_name + " does not exist."
```

Comment:
since we don't use this style maybe we say something like

"Project ____ for user ____ does not exist"